Sources/PubNub/EventEngine/Subscribe/Helpers/SubscribeInput.swift

```diff
@@ -120,21 +120,21 @@ struct SubscribeInput: Equatable {
   }
   
   func removing(
-    channels: [PubNubChannel],
+    mainChannels: [PubNubChannel],
     presenceChannelsOnly: [PubNubChannel],
-    groups: [PubNubChannel],
+    mainGroups: [PubNubChannel],
     presenceGroupsOnly: [PubNubChannel]
```

Comment:
This could be as simple as:
```swift
  func removing(
    channels: [PubNubChannel],
    presenceChannels: [PubNubChannel],
    groups: [PubNubChannel],
    presenceGroups: [PubNubChannel]
```
Also, should there be `PubNubChannelGroup` instead of `PubNubChannel`?

Reply:
Unfortunately, there's one struct representing channel or channel group. This struct was even exposed as public and its instances are parameters for[ local events](https://github.com/pubnub/swift/blob/master/Sources/PubNub/Events/Subscription/SubscriptionStream.swift#L17-L21). The way we may consider reworking it is:

1. Removing `[PubNubChannel]` from these local events and replace it with other type or even raw `[String]`
2. Provide different type representing a channel group. This type should be used internally for our implementation

This might be a breaking change for someone who uses local events. On the other hand, we don't plan to support them anymore. So if I get the approval, I can introduce these changes after prior consultation with you.